Bukkit-Patches/0004-Add-tps-api-and-tps-command.patch

```diff
@@ -0,0 +1,127 @@
+From 603a9aa325208f7b3630a143f8ba84a636353bcb Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Jakub=20Staro=C5=84?= <kubastaron@hotmail.com>
+Date: Wed, 11 Mar 2020 18:08:34 +0100
+Subject: [PATCH] Add tps api and /tps command
+
+
+diff --git a/src/main/java/canyonmodded/command/TpsCommand.java b/src/main/java/canyonmodded/command/TpsCommand.java
+new file mode 100644
+index 00000000..c7bebdc6
+--- /dev/null
++++ b/src/main/java/canyonmodded/command/TpsCommand.java
+@@ -0,0 +1,49 @@
++package canyonmodded.command;
```

Comment:
Wrong package name

Reply:
What else should it be?

Reply:
`com.canyonmodded.command`, just like the server patches

Reply:
Oo, I forgot `com`, i will fix that